### ISSUE TITLE
Fatal error when --directory path has an em-dash

### ISSUE THREAD

**Issue by 75th** [JamieMason/ImageOptim-CLI#138]
Just got this fatal error, which was fixed when I changed the em-dash in a folder name to a hyphen.

```imageoptim -s -a -d /Users/lanny/Desktop/Working\ files\ —\ MKT/Product\ catalog\ stuff/catalog-images
sort: string comparison failed: Illegal byte sequence
sort: Set LC_ALL='C' to work around the problem.
sort: The strings compared were `/USERS/LANNY/DESKTOP/WORKING FILES \302\200\224 MKT/PRODUCT CATALOG STUFF/CATALOG-IMAGES/MKT_PRODUCTCATALOG_3RING-PAGE8.PNG' and `/USERS/LANNY/DESKTOP/WORKING FILES \302\200\224 MKT/PRODUCT CATALOG STUFF/CATALOG-IMAGES/MKT_PRODUCTCATALOG_3RING-PAGE9.PNG'.
Running ImageAlpha...
find: /var/folders/93/qg4vhthx5p3cd4qg_t_j24300000gp/T/imageoptim-cli/png: No such file or directory
Running ImageOptim...
Runtime error (func=(main), adr=9): Divide by zero
(standard_in) 2: parse error```

**Comment by JamieMason** [comment]
Thanks for this @75th and sorry for the delay, I'll look into this.

**Comment by JamieMason** [comment]
Thanks @azusa-tomita, released then in 1.15.3.
